src/site/topics/stacks/stack-adt.rst

```diff
@@ -1,8 +1,8 @@
-*************
-The Stack ADT
+ *************
+The ``Stack`` ADT
 *************
 
-* Stacks are a linear collection of elements
+* ``Stack``s are a linear collection of elements
```

Comment:
check the artifact to see how it renders. I believe you need a backslash before between the backtick and the s (``Stack``\s).